cmd/main_test.go

```diff
@@ -110,38 +111,6 @@ func createNamespace(k8sClient client.Client, name string) (k8sapi.Namespace, er
 	return *namespaced, nil
 }
 
-func deleteRole(k8sClient client.Client, nsName string, roleName string) {
-	role := &rbacv1.Role{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      roleName,
-			Namespace: nsName,
-		},
-	}
-	err := k8sClient.Delete(context.Background(), role)
-	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error deleting the role %s in namespace %s: %v\n", roleName, nsName, err))
-}
-
-func deleteRoleBinding(k8sClient client.Client, nsName string, roleBindingName string) {
-	roleBinding := &rbacv1.RoleBinding{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      roleBindingName,
-			Namespace: nsName,
-		},
-	}
-	err := k8sClient.Delete(context.Background(), roleBinding)
-	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error deleting the role binding %s in namespace %s: %v\n", roleBindingName, nsName, err))
-}
-
-func deleteNamespace(k8sClient client.Client, nsName string) {
-	ns := &k8sapi.Namespace{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: nsName,
-		},
-	}
-	err := k8sClient.Delete(context.Background(), ns)
-	Expect(err).NotTo(HaveOccurred(), fmt.Sprintf("Error deleting the namespace: %s: %v\n", nsName, err))
-}
-
 func performHTTPGetCall(url string, header HTTPheader) (*HTTPResponse, error) {
```

Comment:
Maybe we should add this to `utils` as well

Reply:
When I add to the utils having trouble with the HTTPheader struct.